### DISCUSSION TITLE
Issues with linter-pycodestyle package in Atom

### DISCUSSION THREAD

**Discussion by Agent4580** [community/community#31992] [Actions]
Hello everybody! I recently started my programming journey and switched to Atom. I did everything to make it comfortable to program in Python. But I ran into an error, which is indicated in the screenshot. I turn to you with the hope of finding a solution to the problem. Thanks in advance!
![Uploading Снимок экрана (3).png…]()

**Comment by airtower-luna** [comment]
I'm afraid I can't tell you what's going on there, but I wanted to point out that Atom is in "sunset": https://github.blog/2022-06-08-sunsetting-atom/ The announcement says that repositories will be archived in December, that means all development will cease then.

So you might want to find another editor that works for you instead of fixing the problem. :sweat_smile:
